assets/js/src/newsletters/sending_status.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import Listing from 'listing/listing.jsx';
+import { CronMixin, MailerMixin } from 'newsletters/listings/mixins.jsx';
+
+const columns = [
+  {
+    name: 'subscriber_id',
+    label: MailPoet.I18n.t('subscriber'),
+    sortable: true,
+  },
+  {
+    name: 'status',
+    label: MailPoet.I18n.t('sendingStatus'),
+  },
+  {
+    name: 'failureReason',
+    label: MailPoet.I18n.t('failureReason'),
+  },
+];
+
+const messages = {
+  onNoItemsFound: () => MailPoet.I18n.t('noSendingTaskFound'),
+};
+
+const SendingStatus = (props) => {
+  const newsletterId = props.match.params.id;
+  const [newsletterSubject, setNewsletterSubject] = React.useState('');
+
+  React.useEffect(() => {
+    MailPoet.Ajax.post({
+      api_version: window.mailpoet_api_version,
+      endpoint: 'newsletters',
+      action: 'get',
+      data: {
+        id: newsletterId,
+      },
+    })
+      .done(res => setNewsletterSubject(res.data.subject))
+      .fail((res) => {
+        setNewsletterSubject('');
+        MailPoet.Notice.error(
+          res.errors.map(error => error.message),
+          { scroll: true }
+        );
+      });
+  }, [newsletterId]);
+
+  return (
+    <>
+      <h1>{MailPoet.I18n.t('sendingStatusTitle')}</h1>
+      <StatsLink
+        newsletterId={newsletterId}
+        newsletterSubject={newsletterSubject}
+      />
+      <Listing
+        limit={window.mailpoet_listing_per_page}
+        location={props.location}
+        params={props.match.params}
+        endpoint="sending_task_subscribers"
+        base_url="sending-status/:id"
+        onRenderItem={item => <div><ListingItem {...item} /></div>}
+        getListingItemKey={item => `${item.taskId}-${item.subscriberId}`}
+        columns={columns}
+        messages={messages}
+        auto_refresh
+        sort_by="created_at"
+        afterGetItems={(state) => {
+          MailerMixin.checkMailerStatus(state);
+          CronMixin.checkCronStatus(state);
+        }}
+      />
+    </>
+  );
+};
+SendingStatus.propTypes = {
+  location: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
Do we always have to disable eslint on this line? you know what you need, don't you? Can't you use `shape` instead of `object` here?